docs/source/standard.md

```diff
@@ -39,67 +39,92 @@ What the TRE organisation does to ensure information risk is measured and manage
+| Statement | Guidance | Manditory Status
+| --------- | -------- | ----------- |
+| You have a process in place to ensure any new project requiring a TRE meets relevant legal, ethical and contractual requirements | For example national legislation such as GDPR, discipline specific regulation like GCP or contractural requirements from a specific data provider such as a company or research partner organisation | Mandatory
+| You have a process in place to monitor changes to any legal, ethical and contractual requirements, and to update your policies accordingly | | Mandatory |
 
 ### 1.3 Quality management
 
-The ability of the TRE organisation to measure and control quality of processes, documentation and outputs.
+*The ability of the TRE organisation to measure and control quality of processes, documentation and outputs.*
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Document management
 
-#### 1.4 Audit and reporting
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| All policies & standard operating procedures relevant to the TRE organisation are controlled | This may include measures like restricting edit access to relevant documents, and recording acceptance of policies for all TRE organisation members | Mandatory |
 
-The ability of the TRE organisation to adhere to stated processes and external standards.
+#### Issue management
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have a clear process in place for addressing activity within your TRE organisation that deviates from your policies and standard operating procedures | This can include measures like triage analysis and a process for updating policies | Mandatory |
+| You have methods in place to record progress in resolving issues with, and deviations against, your policies | | Mandatory |
 
-### 1.5 Risk management
+### 1.4 Risk management
 
-The ability of the TRE organisation to measure, forecast and evaluate risks to information.
+*The ability of the TRE organisation to measure, forecast and evaluate risks to information.*
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Risk assessment
+
+| Statement | Guidance | Mandatory Status |
+| --------- | -------- | ---------------- |
+| You have a way to score risk to understand the underlying severity | You have a risk assesment methodology for scoring risks on multiple axes such as impact and likelihood | Mandatory |
+| You have a process for treating risk using additional controls | Risks can be reduced to a level which brings it within agreed levels of appetite | Mandatory |
+| You have an understanding of risk appe*tite | You have an understanding of risk appetite, ownership of risk and ability to accept risk which falls outside of the appetite should that become necessary | Mandatory |
```

Comment:
How does this differ from the first two statements?

Reply:
The key bit seems to be ownership and ability to accept risk outside appetite, not covered by the previous too. Will update wording a little bit